velox/common/tests/BitUtilTest.cpp

```diff
@@ -416,6 +416,43 @@ TEST_F(BitUtilTest, forEachBit) {
   expectedIndex = 3;
   forEachUnsetBit(data, 2, totalBits - 3, calledOnEveryOther);
   EXPECT_EQ(totalBits - 3, expectedIndex);
+
+  auto numEntries = 100;
+  uint32_t size = bits::nwords(numEntries);
+  std::vector<uint64_t> vec(size);
+  uint64_t* vecPtr = vec.data();
+
+  // Test out iterating over all entries when true
```

Comment:
Comments should be full sentences, e.g. end with a period. Please fix throughout.

"all entries when true" sounds strange; how about "Test all bits set." and "Test all bits unset."